external/avro/src/test/scala/org/apache/spark/sql/avro/AvroLogicalTypeSuite.scala

```diff
@@ -349,3 +349,19 @@ class AvroLogicalTypeSuite extends QueryTest with SharedSQLContext with SQLTestU
     }
   }
 }
+
+class AvroV1LogicalTypeSuite extends AvroSuite {
```

Comment:
This should extend `AvroLogicalTypeSuite` instead of `AvroSuite`.

Reply:
Thanks! I have fixed it.